docs/docs-content/clusters/data-center/nutanix/install-pcg.md

```diff
@@ -0,0 +1,217 @@
+
+## Prerequisites
+
+- A Nutanix Prism Central account with *Prism Admin* role.
+
+- A Nutanix Prism Element cluster created.
+
+- A Kubernetes cluster created that has network connectivity with Nutanix Prism. This cluster will connect the Nutanix cloud with Palette via the PCG. Various types of Kubernetes clusters can be used to deploy the PCG. If you need guidance in creating a Nutanix cluster to deploy your PCG, check out the [Set Up Environment and Deploy Cluster](#set-up-environment-and-deploy-cluster) section below, that describes one possible option. 
+
+- A Nutanix subnet created in Nutanix Prism Central that will be assigned to the virtual machines (VMs) deployed in the Kubernetes cluster.
+
+- A Nutanix cloud registered with Palette. For more information, review [Register Nutanix Cloud](register-nutanix-cloud.md).
+
+- A Nutanix Cluster API (CAPI) OS image. For guidance on creating the image, refer to [Building CAPI Images for Nutanix Cloud Platform](https://image-builder.sigs.k8s.io/capi/providers/nutanix.html#building-capi-images-for-nutanix-cloud-platform-ncp).
+
+
+
+
+## Install PCG
+
+Use the following steps to install the PCG in your Kubernetes workload cluster.
+
+1. Log in to [Palette](https://console.spectrocloud.com/).
+
+2. From the left **Main Menu**, select **Tenant Settings**.
+
+3. Next, on the **Tenant Settings Menu**, select **Private Cloud Gateways** and click on **Add New Private Cloud Gateway**.
+
+4. Select **Self Hosted** in the next window that Palette displays.
+
+5. Provide a name for the PCG and use the **drop-down Menu** to select Nutanix as the cloud type. Click **Confirm** to continue. You will be redirected to the Private Cloud Gateway Overview page. 
+
+6. To install the Palette agent, copy the kubectl commands from the slide-out panel and execute them against your workload cluster.
+
+7. Close the slide-out panel when you have copied both commands. The PCG Overview page **Cluster Status** field will display **Pending** while the PCG is deploying. The deployment is complete when the **Cluster Status** field displays the status **Running**.  
+
+
+## Validate
+
+When deployed, the PCG registers itself with Palette. Use the steps below to verify if the PCG registration is successful.
+
+1. Log in to [Palette](https://console.spectrocloud.com/).
+
+
+2. Navigate to the **left Main Menu** and select **Tenant Settings**.
+
+
+3. Next, on the **Tenant Settings Menu**, select **Private Cloud Gateways**.
+
+
+4. Locate the PCG and verify it is installed and in the **Running** state. With the PCG successfully deployed in your Kubernetes workload cluster, you can delete the kind cluster that was used to bootstrap the workload cluster. 
+
+  ```bash
+  kind delete cluster --name pcg-pilot
+  ```
+
+## Set Up Environment and Deploy Cluster
```

Comment:
We need to find a better section title for this, it's too vague currently. Something like "Deploy a Kubernetes cluster to host a PCG"

Reply:
Revised as: Deploy a Kubernetes Cluster to Host the PCG